src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataDeltaFeedSerializer.cs

```diff
@@ -234,13 +241,32 @@ public override async Task WriteObjectAsync(object graph, Type type, ODataMessag
                     }
 
                     lastResource = entry;
-                    IEdmChangedObject edmChangedObject = entry as IEdmChangedObject;
-                    if (edmChangedObject == null)
+                    EdmDeltaEntityKind deltaEntityKind;
+
+                    if (writeContext.IsUntyped)
```

Comment:
Can you explain this change?

Reply:
We've added a check to check if the writeContext is `unTyped`